resources/public/api/conf/3.0/queryParameters.raml

```diff
@@ -37,4 +37,10 @@ traits:
       claimType:
         description: '[This query parameter is only available in Sandbox]<br/><br/>The type of loss claim. Limited to the following possible values: <ul><li><b>`carry-sideways`</b></li></ul>'
         required: false
+  taxYearBroughtForwardFrom:
+      queryParameters:
+        taxYear:
```

Comment:
this needs to be named  as taxYearBroughtForwardFrom or it will show as taxYear on the RAML